### ISSUE TITLE
convert the model to onnx , but BatchNorm2d is lost

### ISSUE THREAD

**Issue by cqray1990** [pytorch/pytorch#93343]
### 🐛 Describe the bug

class Net(nn.Module):
    def __init__(self):
        super(Net, self).__init__()
        self.conv = nn.Conv2d(
            3,
            64,
            3,
            stride=2,
            padding=0,
        )
      
        self.bn = nn.BatchNorm2d(64)
        self.relu = nn.ReLU(inplace=True)

    def forward(self,inx):
        x = self.conv(inx)
        x = self.bn(x)
        x = self.relu(x)

        return x

model = Net()
model = model.cuda()

x = torch.rand([1, 3, 224, 224]).to("cuda")
out = model(x)
print(out)
print(model)
torch.onnx.export(model,
                          x,
                          "./modelr.onnx",
                          export_params=True,
                          verbose=True,
                          opset_version=11,
                          do_constant_folding=True,
                          input_names=["input"], output_names=["output"],
                          )

netron.start("./modelr.onnx")

### Versions

pytorch version : 1.10.1+cuda11.3
onnx 1.10.0
![modelr onnx](https://user-images.githubusercontent.com/32585434/215686221-1430ef5a-e6af-46b8-8c07-3bc9c29c57d7.png)

**Comment by vadimkantorov** [comment]
If it's in eval mode, maybe batchnorm was fused into the Conv weights?

**Comment by cqray1990** [comment]
> 

did it fused into The Conv weights default? but with old version Bn is still exists

**Comment by BowenBao** [comment]
@cqray1990 Yes conv-bn fusion is done by default.

**Comment by BowenBao** [comment]
> @BowenBao maybe onnx export should have a verbose mode where it prints some stats about applied fusions and transforms, or already has such a mode?

We are working on a diagnostics system that tracks and reports these stats.
